README.md

```diff
@@ -1,11 +1,12 @@
 # Unidata AMS 2021 Student Conference PyAOS Workshop
 
-This repository holds material and notebooks for use during the Unidata AMS 2021 Student Conference PyAOS Workshop, as part of the 20th Annual AMS Student Conference.
+This repository holds material and notebooks for use during the [Unidata AMS 2021 Student Conference PyAOS Workshop](https://unidata.github.io/pyaos-ams-2021/index.html), as part of the 20th Annual AMS Student Conference.
 
 ## Agenda
 
-TBA
+Please visit [the workshop agenda page](https://unidata.github.io/pyaos-ams-2021/agenda.html).
 
 ## Acknowledgements
 
-The JupyterHub for this workshop is part of the Unidata Science Gateway (https://doi.org/10.5065/688s-2w73). We thank Andrea Zonca (San Diego Supercomputing Center), Jeremy Fischer (Indiana University), the NSF Jetstream (http://dx.doi.org/10.1145/2792745.2792774) team, and the NSF XSEDE Extended Collaborative Support Service (ECSS) program (https://doi.org/10.1007/978-3-319-32243-8_1) for assistance with this JupyterHub.
+The JupyterHub for this workshop is part of the Unidata Science Gateway (https://doi.org/10.5065/688s-2w73).
+We thank Andrea Zonca (San Diego Supercomputing Center), Jeremy Fischer (Indiana University), the NSF Jetstream (http://dx.doi.org/10.1145/2792745.2792774) team, and the NSF XSEDE Extended Collaborative Support Service (ECSS) program (https://doi.org/10.1007/978-3-319-32243-8_1) for assistance with this JupyterHub.
```

Comment:
Could turn the DOI URLs into markdown links.

Reply:
Done.